teamcity.build

```diff
@@ -13,7 +14,15 @@
 
 	<property name="build.number" value="${CCNetLabel}" if="${property::exists('CCNetLabel')}" />
 
-	<target name="clean-configure-test" depends="cleanall init copy-teamcity-configuration binaries test verify-test-results binaries-zip" />
+	<target name="clean-configure-test" depends="cleanall init copy-teamcity-configuration binaries start-db-service test stop-db-service verify-test-results binaries-zip" />
```

Comment:
This way of doing things will cause the service to stay running if the test step fails. It is not ideal, but it should not cause much issues, provided some other PR succeeds and still ends up stopping the service. (Starting an already started service does not fail.)

I have looked for using the `trycatch` Nant node, but have found out it was requiring many additional changes, so I have not considered it worth of being done:
- Needs upgrading `Nant` to latest version (0.92, we use 0.91)
- Needs rewriting the task as
```xml
<target name="clean-configure-test">
  <trycatch>
    <try>
      <!-- Use a sub-target doing it all instead of putting test prerequisites in
             depends attribute the calling just test target here, otherwise build is
             run twice -->
      <call target="do-clean-configure-test" />
    </try>
    <finally>
      <!-- Use failonerror="false" in order to avoid hiding a failure from the try -->
      <call target="stop-db-service" failonerror="false" />
    </finally>
  </trycatch>
</target>

<target name="do-clean-configure-test" depends="cleanall init copy-teamcity-configuration binaries start-db-service test verify-test-results binaries-zip" />
```
- Needs switching `<xmlpoke>` tasks to `failonerror="false"` due to nant/nant#44 fixed in 0.92, combined with the `test` target calling it on a file missing most of the nodes targeted by the `<xmlpoke>`.